Source/Core/InputCommon/ControllerEmu/ControlGroup/ControlGroup.h

```diff
@@ -79,7 +81,9 @@ class ControlGroup
   const std::string name;
   const std::string ui_name;
   const GroupType type;
+  const bool disablable;
```

Comment:
Maybe thats just me, but this word sounds terrible in my head :/
🚲 Am I alone, wishing this to be something more like `can_be_disabled` instead?

Feel free to ignore this one before we start hitting each other with paint buckets in disagreement over the shed's color.